FrontBackMatter/LateralStatistics.tex

```diff
@@ -85,47 +85,58 @@ \subsection{Co-occurence statistics}
       was trained based on these statistics. B) The sparse set
       of labeled edges extracted from a single image. C) Diagram
       showing how the angular difference $\theta$ and azimuth angle
-      $\phi$ and relative azimuth $\psi$ are computed from two
+      $\varphi$ and relative azimuth $\psi$ are computed from two
       edges. Reproduced from \cite{Perrinet2015}.}
 	\label{classifier}
 \end{figure}
 
 The afferent receptive fields in the visual cortex are usually assumed
-to act as feature detectors (or at least feature-selective units). In
-primary visual cortex connections between these feature detectors
-could therefore represent the co-occurence of the features,
-i.e. second- or higher-order correlations between simple oriented
-edges. In order to test this idea, the models were trained on various
-synthetic and natural image datasets, and the statistics embedded in
-the lateral connections were decoded.
-
-This analysis was done by computing the angles $\theta$, $\phi$, and $\psi$ (as
-defined in figure (C) in Figure~\ref{classifier}) as well as the
-Euclidean distance of the receptive-field centers for each pre- and
-post-synaptic pair of neurons and then binning them weighted by the
-strength of connections between them. The angle $\theta$ was computed
-simply as the difference between the pre- and post-synaptic neurons'
-orientation preference in the orientation map, and the position was
-defined by computing the center of gravity of the neuron's afferent receptive
-fields, allowing the angles $\phi$ and $\psi$ to be calculated. As in
-the previous chapter, the weights were first thresholded, leaving only the
-strongest 10\% to ensure that the model roughly matches the known
-sparse and patchy pattern observed in anatomical tracing
-studies. Additionally, neurons were selected from strong
-iso-orientation regions by selecting the neurons with a local
+to act as feature-selective units. In primary visual cortex,
+connections between these feature detectors could therefore represent
```

Comment:
Really strongly recommend against ever calling something feature detector; that implies a binary response that is just not true.